app/controllers/application_controller.rb

```diff
@@ -1790,6 +1790,17 @@ def task_supported?(typ)
     end
 
     vms = VmOrTemplate.where(:id => vm_ids)
+    if typ == "migrate"
+      # if one of the providers in question cannot support simultaneous migration of his subset of
+      # the selected VMs, we abort
+      if vms.group_by(&:ext_management_system).except(nil).any? do |ems, ems_vms|
+        ems.respond_to?(:supports_migrate_for_all?) && !ems.supports_migrate_for_all?(ems_vms)
+      end
+        add_flash(_("This VMs can not be migrated together."), :error)
```

Comment:
@himdel I guess we could also teach @miq-bot to check grammar in strings of submitted PRs among other things xD